src/imbi_api/endpoints/uploads.py

```
@@ -0,0 +1,312 @@
+        upload_id,
+        auth.user.email,
+        content_type,
+        len(data),
+    )
+
+    return _upload_response(upload_model)
+
+
+@uploads_router.get('/')
+async def list_uploads(
+    auth: typing.Annotated[
```

Ruff (0.14.14)

[warning] Unused function argument: `auth`

(ARG001)